test/orders-test.js

```diff
@@ -1669,3 +1670,340 @@ suite('delete orders', function() {
     .end(done);
   });
 });
+
+suite('get order book', function() {
+  var self = this;
+
+  setup(testutils.setup.bind(self));
+  teardown(testutils.teardown.bind(self));
+
+  test('v1/accounts/:account/order_book/:base/:counter', function(done) {
+    var firstMessageId;
+
+    self.wss.on('request_ledger', function(message, conn) {
+      assert.strictEqual(message.ledger_index, 'validated');
+      conn.send(fixtures.requestLedgerResponse(message, {
+        ledger: LEDGER
+      }));
+    });
+
+    self.wss.on('request_book_offers', function(message, conn) {
+      firstMessageId = firstMessageId || message.id;
+
+      assert.strictEqual(message.command, 'book_offers');
+      assert.strictEqual(message.ledger_index, LEDGER);
+      assert.strictEqual(message.taker, addresses.VALID);
+
+      if (message.id === firstMessageId) {
+        // Bids
+        conn.send(fixtures.requestBookOffersBidsResponse(message));
+      } else {
+        // Asks
+        conn.send(fixtures.requestBookOffersAsksResponse(message));
+      }
+    });
+
+    self.app
+    .get('/v1/accounts/' + addresses.VALID + '/order_book/' + 'BTC+' + addresses.ISSUER + '/USD+' + addresses.ISSUER)
+    .expect(testutils.checkStatus(200))
+    .expect(testutils.checkHeaders)
+    .expect(testutils.checkBody(fixtures.RESTOrderBookResponse({
+      ledger: LEDGER
+    })))
+    .end(done);
+  });
+
+  test('v1/accounts/:account/order_book/:base/:counter -- with partially funded ask', function(done) {
+    var firstMessageId;
+
+    self.wss.on('request_ledger', function(message, conn) {
+      assert.strictEqual(message.ledger_index, 'validated');
+      conn.send(fixtures.requestLedgerResponse(message, {
+        ledger: LEDGER
+      }));
+    });
+
+    self.wss.on('request_book_offers', function(message, conn) {
+      firstMessageId = firstMessageId || message.id;
+
+      assert.strictEqual(message.command, 'book_offers');
+      assert.strictEqual(message.ledger_index, LEDGER);
+
+      if (message.id === firstMessageId) {
+        // Bids
+        conn.send(fixtures.requestBookOffersBidsResponse(message));
+      } else {
+        // Asks
+        conn.send(fixtures.requestBookOffersAsksPartialFundedResponse(message));
+      }
+    });
+
+    self.app
+    .get('/v1/accounts/' + addresses.VALID + '/order_book/' + 'BTC+' + addresses.ISSUER + '/USD+' + addresses.ISSUER)
```

Comment:
Not a huge deal but `'/order_book/' + 'BTC+'` should just be `'/order_book/BTC+'`